frame/balances/src/lib.rs

```diff
@@ -194,7 +195,7 @@ pub mod pallet {
 	pub trait Config<I: 'static = ()>: frame_system::Config {
 		/// The balance of an account.
 		type Balance: Parameter + Member + AtLeast32BitUnsigned + Codec + Default + Copy +
-			MaybeSerializeDeserialize + Debug + MaxEncodedLen;
+			MaybeSerializeDeserialize + Debug + MaxEncodedLen + TypeInfo;
```

Comment:
as a future thought, maybe we want to create a new trait `FrameType` which includes both `MaxEncodedLen` and `TypeInfo`, and anything else in the future.

Reply:
currently there is Parameter which bounds everything needed to be in a call argument.
Maybe we should create another trait for stuff needed to be stored.

Also Parameter is I think currently not very self-explanatory name